ui/apps/everest/src/components/cluster-form/advanced-configuration/advanced-configuration.tsx

```diff
@@ -68,6 +68,7 @@ export const AdvancedConfigurationForm = ({
       );
     }
   }, [clusterInfo]);
+
   const handleBlur = (value: string, fieldName: string, hasError: boolean) => {
     if (!hasError && !value.includes('/') && value !== '') {
       setValue(fieldName, `${value}/32`);
```

Comment:
@Tikdev00 please add `{ shouldValidate: true }` as third argument here, or IP addresses without netmask won't fail the validation (e.g. "192.168.1.1" when there's already "192.168.1.1/32"